yt_dlp/extractor/mixch.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    traverse_obj,
+)
+
+
+class MixchIE(InfoExtractor):
+    IE_NAME = 'mixch'
+    # allow omitting last /live in the URL, though it's likely uncommon
+    _VALID_URL = r'https?://(?:www\.)?mixch\.tv/u/(?P<id>\d+)'
+
+    TESTS = [{
+        'url': 'https://mixch.tv/u/16137876/live',
+        'skip': 'live has ended',
```

Comment:
either add fields or make it `only_matching`. Though idk why this test is even needed

Reply:
https://github.com/yt-dlp/yt-dlp/pull/866#discussion_r701060313

Reply:
That's what the other test is for, no?

Reply:
oh
converted to only_matching test